app/products.py

```diff
@@ -0,0 +1,114 @@
+"""
+ Must have the environment variables set properly prior to running the script
+"""
+from woocommerce import API
+import json
+import os
+from faker import Faker
+import random
+
+fake = Faker()
+
+
+def create_api_object():
+    wcapi = API(
+        url=os.environ.get('BASE_URL'),
+        consumer_key=os.environ.get('API_KEY'),
+        consumer_secret=os.environ.get('API_SECRET'),
+        version="wc/v3",
+        timeout=15
+    )
+    return wcapi
+
+
+store_api = create_api_object()
+
+
+class Products:
+    def __init__(self) -> None:
+        self.endpoint = 'products'
+        self.products = self.fetch_all_products()
+
+    def create_random_product(self):
+        product_data = self.generate_random_product_data()
+
+        try:
+            response = store_api.post(self.endpoint, data=product_data)
+            if response.ok:
```

Comment:
I've never seen `response.ok`. Normally you would check the status code `response.status_code == 201`. 
If this works fine but I wonder if this true for all REST Api or is it unique to woocommerce api.